src/vmware/DEVELOPMENT.md

```diff
@@ -5,7 +5,7 @@
 ```powershell
 rm ..\azure-cli-extensions\src\vmware\azext_vmware\vendored_sdks -Recurse
 
-autorest --python --output-folder=..\azure-cli-extensions\src\vmware\azext_vmware\vendored_sdks --use=@autorest/python@5.9.3 --tag=package-2021-06-01 --azure-arm=true --namespace=avs_client --override-client-name=AVSClient specification\vmware\resource-manager\readme.md
+autorest --python --output-folder=..\azure-cli-extensions\src\vmware\azext_vmware\vendored_sdks\avs_client --use=@autorest/python@5.8.0 --tag=package-2022-05-01 --azure-arm=true --namespace=avs_client --override-client-name=AVSClient specification\vmware\resource-manager\readme.md
```

Comment:
What was reason for downgrading from 5.9.3 to 5.8.0?

Reply:
Hi @jonathanhe-msft, next time you can try our new CodeGen tool [aaz-dev-tools](https://github.com/Azure/aaz-dev-tools). It's not relied on Python SDK and very easy to use.

Reply:
I wasn't able to get a successful generation using the 5.9.3. I forget exactly what the error was.